multiversx_sdk/entrypoints/entrypoints.py

```diff
@@ -118,45 +141,81 @@ def get_transaction(self, tx_hash: Union[str, bytes]) -> TransactionOnNetwork:
 
     def create_network_provider(
         self,
-    ) -> Union[ApiNetworkProvider, ProxyNetworkProvider]:
+    ) -> INetworkProvider:
```

Comment:
:+1: 

Theoretical breaking change (output type becomes more abstract). This one is never breaking in practice.